### ISSUE TITLE
[Feature] print instruction for using `jiq` when no stdin is piped in

### ISSUE THREAD

**Issue by aesyondu** [fiatjaf/jiq#18]
What if when one launches `jiq` without receiving anything from stdin, it would print an instruction that it expects a json structure, and finally Ctrl+D to terminate the json string?

```shell
$ jiq
Enter a json formatted string, press Enter and Ctrl+D to continue:
[1,2,3]
^D

[Filter]> .
[
  1,
  2,
  3
]
```

What say you?

**Comment by fiatjaf** [comment]
Yes, that would be good. I am confused sometimes by the lack of response when jiq is launched without anything on stdin.
